### PR TITLE
fix(webhooks): updated content #10045

### PR DESCRIPTION
**Pull request by thekeogh**
ref https://github.com/screencloud/apps/issues/10045

### CODE REVIEW

File: docs/content/webhooks-app.md [hunk 1]
```diff
@@ -92,27 +98,125 @@ Below are examples showcasing various display formats your webhooks will adopt w
...
 
 ![title only post](./images/hook-title-only.png)
 
+The example below demonstrates the simplest data structure you would `POST` to achieve the above result.
+
+```json
+{
+  "items": [
+    {
+      "author": {
```
Comment (shannen-smyth) [inline docs/content/webhooks-app.md:145]: For title only do we remove the author too for demo purposes?

Comment (thekeogh) [reply docs/content/webhooks-app.md:145]: The author is in all the images, so to be consistent with the image I kept the author there. Author is in all the images in fact.

Comment (shannen-smyth) [reply docs/content/webhooks-app.md:145]: Sorry that's what I mean maybe we should create images without the author to show the real bare minimum for the webhook item. It doesn't need an author but might read to the user they do in this case?

Comment (thekeogh) [reply docs/content/webhooks-app.md]: Yep, I agree. I have altered the images now and updated the content. The author is now only on the full content one.

**Review by shannen-smyth — APPROVED** [review]

**Pull request merged**